Library/Formula/smartmontools.rb

```diff
@@ -3,8 +3,8 @@
 class Smartmontools < Formula
   desc "SMART hard drive monitoring"
   homepage 'http://sourceforge.net/apps/trac/smartmontools/'
```

Comment:
Correct homepage URL is http://smartmontools.org/. Sourceforge is used only as SVN server and file storage.